packages/databyss-services/pages/reducer.js

```diff
@@ -36,12 +37,23 @@ export default (state, action) => {
         ..._state,
       }
     }
+    case REMOVE_PAGE_FROM_CACHE: {
+      const _cache = state.cache
```

Comment:
Here we need to do a deepClone because we're mutating the state with the `delete` statement. This will change once we move to `immer`